saffron/e2e-test.sh

```diff
@@ -11,8 +11,34 @@ SRS_ARG=""
 if [ $# -eq 2 ]; then
    SRS_ARG="--srs-filepath $2"
 fi
+
 ENCODED_FILE="${INPUT_FILE%.*}.bin"
 DECODED_FILE="${INPUT_FILE%.*}-decoded${INPUT_FILE##*.}"
```

Comment:
Missing the dot also.
```suggestion
DECODED_FILE="${INPUT_FILE%.*}-decoded.${INPUT_FILE##*.}"
```